protocol/x/vault/keeper/params_test.go

```diff
@@ -46,3 +48,33 @@ func TestGetSetParams(t *testing.T) {
 	require.Error(t, err)
 	require.Equal(t, newParams, k.GetParams(ctx))
 }
+
+func TestGetSetVaultParams(t *testing.T) {
```

Comment:
nit: Test setting params for non-existent vault.

Reply:
Could you clarify what this means? setting params for a vault that has no shares?